instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/internal/LogEventMapperTest.java

```diff
@@ -30,62 +32,62 @@ class LogEventMapperTest {
   @Test
   void testDefault() {
     // given
-    LogEventMapper<Map<String, String>> mapper =
+    LogEventMapper<Map<String, Object>> mapper =
         new LogEventMapper<>(
             ContextDataAccessorImpl.INSTANCE, false, false, false, false, emptyList());
-    Map<String, String> contextData = new HashMap<>();
+    Map<String, Object> contextData = new HashMap<>();
     contextData.put("key1", "value1");
     contextData.put("key2", "value2");
-    AttributesBuilder attributes = Attributes.builder();
+    ExtendedAttributesBuilder attributes = ExtendedAttributes.builder();
```

Comment:
could you elaborate why you didn't just continue using `AttributesBuilder` here?

Reply:
That's only because `captureContextDataAttributes` now takes `ExtendedAttributesBuilder` instead of `AttributesBuilder`. The idea is that I always use `ExtendedAttributes` in all methods of `LogEventMapper`, and convert to `Attributes` at the end if needed.

I figured I could do this because this does not seem to be public API, right?